gnocchi/opts.py

```diff
@@ -193,6 +193,13 @@ def list_opts():
                        default=10, min=0,
                        help='Number of seconds before timeout when attempting '
                             'to do some operations.'),
+            cfg.BoolOpt('group_measures_using_history', default=False,
```

Comment:
I find it always weird to control API behaviour via global config options.
Why isn't that a only flag passed to the API?

Reply:
This was done to avoid changing every system that uses this API to get the new behavior, so, instread of changing lots of systems to use the new API flag, we just set the configuration in gnocchi, and if some system for some reason need the old behavior, it can simple override the config with the API flag.

Reply:
This is not a good approach as an API should never ever change and break existing system.

Reply:
I will remove the config